librz/bin/format/mach0/mach0_defines.h

```diff
@@ -1519,6 +1520,22 @@ struct dyld_chained_ptr_arm64e_auth_bind {
 		auth : 1; // == 1
 };
 
+struct dyld_chained_ptr_64_rebase {
+	uint64_t target : 36,
+		high8 : 8,
+		reserved : 7,
+		next : 12,
+		bind : 1; // == 0
+};
+
+struct dyld_chained_ptr_64_bind {
+	uint64_t ordinal : 24,
+		addend : 8,
+		reserved : 19,
+		next : 12,
+		bind : 1; // == 1
```

Comment:
probably these needs to be parsed with endianness in mind.

Reply:
Judging from my recent experiments with endianness and bitfields on tree-sitter there is actually a chance this might work as-is, but I am not sure yet. I would still leave them as-is right now and convert later, along with all the other structs, because these are the official definitions from dyld code.

Reply:
ok

Reply:
Ok, just checked and it does not work on big endian. But I would still like to fix it later.